src/electron/platform/android/rule-information-provider.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RuleInformation } from './rule-information';
+import { RuleResultsData } from './scan-results';
+
+export class RuleInformationProvider {
+    private supportedRules: RuleInformation[];
+
+    constructor() {
+        this.supportedRules = [];
+        this.supportedRules.push(
```

Comment:
You may be able to initialize the variable explicitly without any pushes:
```
rules = [ new X(), new Y(), new Z() ]; // with new lines
```

Reply:
Used a dictionary, as we discussed